src/others/file_explorer/lv_file_explorer.h

```diff
@@ -0,0 +1,184 @@
+/**
+ * @file lv_file_explorer.h
+ *
+ */
+
+#ifndef LV_FILE_EXPLORER_H
+#define LV_FILE_EXPLORER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../../lvgl.h"
+
+#if LV_USE_FILE_EXPLORER != 0
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef enum {
+    LV_EXPLORER_SORT_NONE,
+    LV_EXPLORER_SORT_KIND,
+} lv_file_explorer_sort_t;
```

Comment:
Did you consider using a callback function for sort?
This would allow the users to apply any custom sort they like, not only "None" and "Kind".

Reply:
This is a good suggestion, but I don't think you need to use callback functions, so I provide [lv_example_file_explorer_3](https://github.com/lvgl/lvgl/pull/3601/commits/50965f7305de0695cce89351833e13febd7ffa3d).